Dockerfile

```diff
@@ -218,8 +161,16 @@ COPY --from=tsbuild /usr/src/app/dist dist
 # TODO: remove in v20
 COPY --from=tsbuild /usr/src/app/node_modules node_modules
 
-ENTRYPOINT ["node", "/usr/src/app/dist/renovate.js"]
-CMD []
+
+COPY bin/ /usr/local/bin/
+
+RUN set -ex; \
+  ln -sf /usr/src/app/dist/renovate.js /usr/local/bin/renovate; \
+  renovate --version;
+
+ENTRYPOINT ["docker-entrypoint.sh"]
```

Comment:
This isn't inherited from buildpack?

Reply:
nope, I've added some sugar to use the image more easily

Reply:
Documented?

Reply:
A little bit, see readme changes. I'll add samples for gitlab and jenkins